spec/requests/content_blocks_spec.rb

```diff
@@ -4,17 +4,34 @@
   let!(:content_block) { create(:content_block) }
   let!(:credential_pair) { create(:credential_pair, api_key: content_block.api_key) }
 
+  # rubocop:disable Metrics/AbcSize
+  def compare_content_blocks(object, content_block)
```

Comment:
hmm, good point. i think i only need it in this file though, so an own matcher would be an overkill?